blueprint/core/blueprint_EcmascriptEngine.cpp

```diff
@@ -119,12 +119,14 @@ namespace blueprint
             return true;
        }
 
+/*
        static void dumpContext(duk_context* ctx)
```

Comment:
One of our build variants shouted about an unused method. Could [maybe_unused] but I believe blueprint is currently using C++11 as minimum standard?

Reply:
Could just as easily remove the function etc.

Reply:
Ohh man, that's a bummer. I think making blueprint available back to C++14/11 would maybe help user adoption but I _do_ often reach for C++17 features. What do you think is better here?